content/waku/2024-gantt.md

```diff
@@ -24,92 +24,126 @@ gantt
-    section test eng 1
-        Peer and connection management tests: sim-conn-mgmt, 2024-05-13, 4w
-        (simulation) Functionality and stress test store v3: sim-storev3, after sim-conn-mgmt, 4w
-        %% This is Store Service Upgrade - item (2) in DST simulation - start with small scale to get faster results
-        (simulation) Compare store topologies: sim-store-cmp, after sim-storev3, 6w
-        (simulation) relay reliability performance impact: sim-relay-rel, after sim-store-cmp sim-req-res rel-relay-g rel-relay-n, 4w
-        (simulation) req-res reliability performance impact: sim-reqres-rel, after sim-relay-rel rel-reqres-g, 6w
-    section research eng 1
-        End-to-end reliability protocol - PoC: crit, e2e-rel-r, 2024-05-23, 20w
-    section research eng 2
+    section Chat.plopezlpz
+        (testing) Custom shard impl of Communities: test-custom-shard, 2024-08-26, 2024-09-30
+        Setup Waku Community on dedicated shard with pre-shared key dos protection: waku-com, after test-custom-shard, 2w
+    section Chat.kaichaosun
+        Review MVDS usage and fail path: crit, mvds, 2024-08-01, 2024-09-30
+        Minimal Community Specification and Implementation: min-com-spec, after mvds, 12w
+    section Go.richard-ramos
+        Implement RLN smart contract for paid, multilevel memberships: rln-sc, 2024-09-01, 6w
+        %% TODO: Estimate
+        Nwaku in Status Desktop: nwaku-status-desktop-g, after rln-sc-d, 12w
+        Status Support: status-support-rr, 2024-09-01, 2024-12-31
+    section Go.chaitanyaprem
+        DoS Protection for Req-Res Protocols: crit, dosreqres-g, 2024-08-01, 2024-09-15
+        Sharding peer mgmt and discovery hardening: crit, sh-peer-mgmt-g, 2024-08-26, 2024-09-12
+        Review usage of content topics in Status Chat and Communities protocol: cont-topic-r, after sh-peer-mgmt-g, 10w
+    section EcoDev.vpavlin
+        (telemetry) direct message reliability: telem-d-msg-rel-1, 2024-08-26, 2024-09-25
+        (telemetry) Measure Bandwidth: telem-bandwidth, after telem-d-msg-rel-1, 8w
+        %% TODO: do bandwidth telemetry?
+        E2e reliability protocol Status integration: crit, e2e-rel-status-g, after e2e-rel-r, 6w
+        %% TODO: fit that properly
+        Specify de-MLS over Waku: demls-spec, 2024-10-01, 4w
+        Co-design sessions: codesign-bd, after cust-int, 2024-11-15
+    section Go.adklempner
+        (telemetry) direct message reliability: telem-d-msg-rel-2, 2024-08-01, 2024-09-25
+        (telemetry) Multicast message reliability: telem-m-msg-rel, after telem-d-msg-rel-2, 4w
+    section Test.stubbsta
+        %% TODO Review estimate
+        (simulation) Functionality and stress test store v3: sim-storev3, 2024-08-01, 8w
+        (simulation) relay reliability performance impact: sim-relay-rel, after sim-store-cmp sim-req-res, 4w
+        (simulation) req-res reliability performance impact: sim-reqres-rel, after sim-relay-rel, 6w
```

Comment:
This still seems mostly accurate except that we'll need a (significant) separate item for Store Sync. Reliability performance testing could also probably be more general.

I'd change it to:
```
 (simulation) Functionality and stress test store v3: sim-storev3, 2024-07-08, 8w
 (simulation) Functionality and stress test store v3: sim-storesync, 2024-09-01, 10w
 (simulation) Reliability performance impact: sim-rel, after sim-storesync, 10w
```